src/Employer/Employer.Web/ViewModels/Part1/Title/TitleViewModel.cs

```diff
@@ -14,7 +21,7 @@ public class TitleViewModel : TitleEditModel
         public string FormPostRouteName => VacancyId.HasValue ? RouteNames.Title_Post : RouteNames.CreateVacancy_Post;
         public PartOnePageInfoViewModel PageInfo { get; set; }
         public bool HasCloneableVacancies { get; set; }
-        public string BackLink =>
-            HasCloneableVacancies ? RouteNames.Dashboard_Index_Get : RouteNames.CreateVacancyOptions_Get;
+        public string BackLinkText { get; set; }
+        public string BackLinkRoute { get; set; }
```

Comment:
I don't think you are taking into account the hasCloneableVacancies logic that you deleted from the original BackLink in the view model. This also needs to be taken into account the in SetBackLinkRoute() function.